### PR TITLE
feat(chart): Added  SriLanka country map

### PR DESCRIPTION
**Pull request by rukshn**
### SUMMARY
The Sri Lanka country map is not included in the chart country map plugin. This commit adds Sri Lanka map to the chart country map plugin.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

image

### TESTING INSTRUCTIONS
```
cd superset/superset-frontend
npm run dev-server
```

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [x] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts [hunk 1]
```diff
@@ -174,6 +175,7 @@ export const countries = {
   slovenia,
   spain,
   sweden,
+  srilanka,
```
Comment (rusackas) [inline superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts:178]: The linter seems to be complaining about this PR. It _might_ be that this is not quite in alphabetical order. You can have it (hopefully) auto-fix by running `pre-commit run --all-files`

**Comment by rusackas** [comment]
The test environment isn't working for some reason. Sigh. 

The pre-commit hook is complaining about formatting. I made a comment above about a hunch I have there. If you could run that and commit the changes, it should allow me to spin up another ephemeral environment for testing. 🤞

File: superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts (hunk 1 above)
Comment (rukshn) [reply superset-frontend/plugins/legacy-plugin-chart-country-map/src/countries.ts:178]: yes i will run it and push again.

**Comment by rukshn** [comment]
> The test environment isn't working for some reason. Sigh.
> 
> The pre-commit hook is complaining about formatting. I made a comment above about a hunch I have there. If you could run that and commit the changes, it should allow me to spin up another ephemeral environment for testing. 🤞

I have fixed the linting issues, the tests should pass now.

**Comment by rukshn** [comment]
@rusackas I think the PR is good to merge

**Review by villebro — APPROVED** [review]

**Comment by rusackas** [comment]
I'm trying to use this now, and I can't seem to get it to work. The ISO 3166-2 codes don't seem to map to the values I'm using, whether they're provinces (LK-1 through LK-9) or Districts (LK-11 through LK-92). Can you share your sample data that's rendering on the map?

Also,  I'm curious if we should change the country name from "Srilanka" to "Sri Lanka"
image

**Pull request merged**